run_dir/static/js/base.js

```diff
@@ -183,6 +183,10 @@ marked.setOptions({
   smartLists: true,
   smartypants: false
 });
+//Fix for the whole text getting inserted as heading ids and messing up the headings
+marked.Renderer.prototype.heading =function(text,level,raw){
```

Comment:
raw argument is not used. Does it have to be there? I have no idea

Reply:
I just copied and overrode the existing function in `marked.min.js`. Not sure if it'll work without it.

Reply:
Ok let's keep it then